apps/codecov-api/graphs/views.py

```diff
@@ -413,7 +413,7 @@ def get_pull_flare(self, pullid):
         pull = Pull.objects.filter(pullid=pullid, repository_id=repo.repoid).first()
         if pull is not None:
             # pull found
-            if pull._flare is not None or pull._flare_storage_path is not None:
+            if pull._flare or pull._flare_storage_path is not None:
```

Comment:
Can `_flare_storage_path` be an empty string? If not then we can simplify it:

```suggestion
            if pull._flare or pull._flare_storage_path:
```